grpcx.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activesupport', '>= 5.0'
   s.add_dependency 'grpc', '>= 1.8.0'
+  s.add_dependency 'grpclb', '~> 0.4.4'
```

Comment:
Please, double check version constraint... It's not `1.x.x` yet, so probs even worth making a single-version constraint.

Reply:
please remove, I don't want to embed this